plugin/storage/cassandra/spanstore/reader_test.go

```diff
@@ -368,7 +391,11 @@ func TestSpanReaderFindTraces(t *testing.T) {
 					assert.EqualError(t, err, testCase.expectedError)
 				}
 				for _, expectedLog := range testCase.expectedLogs {
-					assert.True(t, strings.Contains(r.logBuffer.String(), expectedLog), "Log must contain %s, but was %s", expectedLog, r.logBuffer.String())
+					assert.True(t,
```

Comment:
there is assert.Contains()

Reply:
which also makes the last 3 args unnecessary since it will show the diffs automatically